src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -78,10 +78,10 @@ public async Task<ActionResult<MethodologyVersionViewModel>> GetMethodology(Guid
         }
 
         [HttpGet("publication/{publicationId:guid}/methodologies")]
-        public async Task<ActionResult<List<MethodologyVersionSummaryViewModel>>> ListLatestMethodologyVersions(Guid publicationId)
+        public async Task<ActionResult<List<MethodologyVersionSummaryViewModel>>> ListLatestApprovedMethodologyVersions(Guid publicationId)
         {
             return await _methodologyService
-                .ListLatestMethodologyVersions(publicationId)
+                .ListLatestApprovedMethodologyVersions(publicationId)
                 .HandleFailuresOrOk();
```

Comment:
Hang on, won't this introduce a new bug where an analyst that should be able to see draft methodologies on the 'Manage methodologies' tab won't see them now?